cluster/vagrant-openshift/setup_master.sh

```diff
@@ -37,9 +37,11 @@ done
 openshift_ansible_dir="/root/openshift-ansible"
 inventory_file="/root/inventory"
 
-# Clone openshift-ansible repository and apply fix for https://github.com/openshift/openshift-ansible/issues/6756
 mkdir -p /root/openshift-ansible
 git clone https://github.com/openshift/openshift-ansible.git $openshift_ansible_dir
```

Comment:
you can add `-b openshift-ansible-3.9.0-0.40.0` to git-cone command to checkout specific branch.

Reply:
fixed